src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2456,6 +2469,7 @@ private void readInput() {
     long lastJavaSourceChange;
     long lastJavaTestChange;
     Map<File, Long> lastBuildFileChange;
+    boolean triggerAppRefresh;
```

Comment:
I think maybe this was deleted since mine but you added it back in?

Reply:
Good catch.